vncdotool/client.py

```diff
@@ -243,7 +244,15 @@ def _capture(self, filename, *args):
         self.deferred.addCallback(self._captureSave, filename, *args)
         return self.deferred
 
+    def framebufferUpdateRequest(self, *args):
+        self.screen_updated = False
```

Comment:
why do this in framebufferUpdateRequest() rather than _capture()?

Reply:
Basically, because in the future I might want to use the PIL Image object directly without writing it out to disk, so I didn't want to couple it to actually saving the screenshot.